docs/docs/en/guide/task/spark.md

```diff
@@ -15,34 +15,33 @@ Spark task type for executing Spark application. When executing the Spark task,
+|----------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Node Name                  | Set the name of the task. Node names within a workflow definition are unique.                                                                                                                                                 |
+| Run flag                   | Indicates whether the node can be scheduled normally. If it is not necessary to execute, you can turn on the prohibiting execution switch.                                                                                    |
+| Description                | Describes the function of this node.                                                                                                                                                                                          |
+| Task priority              | When the number of worker threads is insufficient, they are executed in order from high to low according to the priority, and they are executed according to the first-in, first-out principle when the priority is the same. |
+| Worker group               | The task is assigned to the machines in the worker group for execution. If Default is selected, a worker machine will be randomly selected for execution.                                                                     |
+| Task group name            | The group in Resources, if not configured, it will not be used.                                                                                                                                                               |
+| Environment Name           | Configure the environment in which to run the script.                                                                                                                                                                         |
+| Number of failed retries   | The number of times the task is resubmitted after failure. It supports drop-down and manual filling.                                                                                                                          |
+| Failure Retry Interval     | The time interval for resubmitting the task if the task fails. It supports drop-down and manual filling.                                                                                                                      |
+| Timeout alarm              | Check Timeout Alarm and Timeout Failure. When the task exceeds the "timeout duration", an alarm email will be sent and the task execution will fail.                                                                          |
```

Comment:
It is seems we already change the general parameter into separate parameter files, can you rebase to latest and take a look?

Reply:
Sure.